charts/nirmata/README.md

```diff
@@ -8,25 +8,68 @@
 - View policy enforcement as events.
 - Scan existing resources for violations.
 
-Access the complete user documentation and guides at: https://kyverno.io.
+Access the complete user documentation and guides at: <https://kyverno.io>.
 
 ## Introduction
 
 This chart bootstraps a Kyverno deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
 
+## Prerequisite
+### Get token from Nirmata support
+You need the token to download the images. Please contact <support@nirmata.com> to get the token.
+
+### Install kyverno operator
+
+#### 1. Add Kyverno Helm Repository
+```console
+helm repo add nirmata https://nirmata.github.io/kyverno-charts/
+helm repo update nirmata
+```
+
+#### 2. (Optional) If a custom CA is used, create a configmap corresponding to the same with key custom-ca.pem. E.g.
+Create the namespace
+```console
+kubectl create namespace nirmata-kyverno-operator
```

Comment:
We can remove this step and add --create-namespace in the following install command if not present